src/ssl.c

```diff
@@ -5424,8 +5422,16 @@ int wolfSSL_CertManagerEnableOCSPStapling(WOLFSSL_CERT_MANAGER* cm)
 
     return ret;
 }
-
-
+#if defined(SESSION_CERTS)
+STACK_OF(WOLFSSL_X509)* wolfSSL_get_peer_cert_chain(const WOLFSSL* ssl)
```

Comment:
It suppose to return ssl->session.chain, but it would be a bit tricky if we want to get the value
in api.c without including "internal.h". I assumed the code in SSL_get_peer_cert_chain is simple, so we can assume it returns expected value if it is not NULL.

Could you give some idea how you usually work on this kind of verification?